### ISSUE TITLE
Catching up to tesseract_ros

### ISSUE THREAD

**Issue by marrts** [tesseract-robotics/tesseract_ros2#34]
With #33 most of the functionality is there, but there are a few differences or changes that need to be made to fully catch up.

- [ ] Topic fields in the workbench don't actually look up ROS topics, they just take string inputs
- [ ] Can't run waitForInput() when running from a launch because of how ROS 2 interfaces with the terminal, an alternative should be proposed, or we can just choose to drop this
- [ ] Manipulation widget does not work in the workbench, for some reason we are unable to move around the manipulation objects, but we can change numbers to move the robot to different positions, it is just less efficient
- [ ] `tesseract_rviz` causing a build issue when rebuilding
- [ ] If you want to load the URDF/SRDF into RVIZ as parameters you need to launch RVIZ node with those parameters declared, again this is more of a difference between ROS and ROS 2 than a tesseract thing
- [ ] Currently tesseract_rviz uses `boost::shared_ptrs` everywhere because that's what ROS 1 rviz did, but ROS 2 uses `std::shared_ptrs` so we should probably update to match
- [ ] The planning server itself isn't yet ported

**Comment by smith-doug** [comment]
Here https://github.com/marrts/tesseract_ros2/pull/1

**Comment by Levi-Armstrong** [comment]
@marrts can this be closed?

**Comment by rjoomen** [comment]
No, some issues are still open, I think. I suggest to check the points that are done though.

**Comment by marrts** [comment]
Checked off the issues resolved by @rjoomen

**Comment by rjoomen** [comment]
@Levi-Armstrong I think the point about the manipulation widget can be closed, as it seems to work the same as tesseract_ros1 now, right? Then we could close the entire issue, as the other two points can't be fixed.

**Comment by Levi-Armstrong** [comment]
Sound good.
